test/alternator/test_returnconsumedcapacity.py

```diff
@@ -334,6 +334,25 @@ def test_simple_batch_get_items(test_table_sb):
     assert response['ConsumedCapacity'][0]['TableName'] == test_table_sb.name
     assert 2 == response['ConsumedCapacity'][0]['CapacityUnits']
 
+# This test reproduces a bug where the consumed capacity was divided by 16 KB,
+# instead of 4 KB. The total consumed capacity should be 22 RCU (17 for the
+# large item and 5 for the small item).
+def test_batch_get_items_large(test_table_sb):
+    p1 = random_string()
+    c1 = random_bytes()
+    test_table_sb.put_item(Item={'p': p1, 'c': c1, 'a': 'a' * 64 * KB})
```

Comment:
Can you add a comment that explains why this is 17 RCU, the same goes for the one below that explains why it's 5